### DISCUSSION TITLE
JPEG XL / JXL in the AppImage

### DISCUSSION THREAD

**Discussion by ChristianMayer** [ImageMagick/ImageMagick#6993] [Development]
### ImageMagick version

7.1.1-25

### Operating system, version and so on

Linux (Ubuntu 22.04.3 LTS)

### Description

I just downloaded `magic` from the homepage (https://imagemagick.org/script/download.php) as Ubuntu delivers only a very old version and I wanted to use JPEG XL / JXL.
But to my surprise `./magick -list format` doesn't list it as being built in.

Can this format please be added?

Thank you!

**Comment by carlosvsilva** [comment]
Facing the same issue, with Ubuntu being a major distro, which doesn't move to IM7 due to new syntax, and JXL being a format propped up by the largest vendors and platforms, I think it's warranted to be compiled in the AppImage. Thanks
